inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -356,7 +356,7 @@
                     self._update_logger.status = OTA_PENDING
                     self._update_logger.error = ""
                 self._update_logger.save_log()
-                if self.sota_mode == 'download-only' or self._device_reboot in ["No", "N", "n", "no", "NO"]:  # pragma: no cover
+                if (self.sota_mode == 'download-only') or (not self._is_reboot_device()):
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** The method execute_from_manifest has been updated to use the new variable name _reboot_device. Ensure that the method's logic correctly interprets the value of this variable, especially in conditions where a reboot decision is made.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                if (self.sota_mode == 'download-only') or (not self._reboot_device):
```